airsonic-main/src/main/java/org/airsonic/player/util/Util.java

```diff
@@ -117,10 +119,11 @@ public static void setContentLength(HttpServletResponse response, long length) {
         return result;
     }
 
-    private static ObjectMapper objectMapper = new ObjectMapper();
-    static {
-        objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
-    }
+    private static ObjectMapper objectMapper = new ObjectMapper()
+            .findAndRegisterModules()
+            .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+            .configure(DeserializationFeature.READ_DATE_TIMESTAMPS_AS_NANOSECONDS, false)
```

Comment:
If I understand correctly, this changes all DB operations to not care about nanoseconds anymore, right? If so, I'm okay with this, we don't gain a lot by storing nanoseconds.

Reply:
Not quite. This objectMapper is actually not used to serialize to the DB.

We seem to be using it for converting MediaLibraryStatistics to a Map<String, String> for storing as commit metadata in the lucene index whenever the scan data is committed. I believe that is a recent addition in #1262 

This is needed primarily for the reason that without this, deserialization reads whole numbers as number of seconds instead of number of milliseconds. Decimals will still be read at nanosecond resolution because that's not ambiguous